src/Language/PureScript/Docs/Prim.hs

```diff
@@ -322,20 +332,46 @@ number :: Declaration
 number = primType "Number" $ T.unlines
   [ "A double precision floating point number (IEEE 754)."
   , ""
-  , "Construct values of this type with literals:"
+  , "Construct values of this type with literals."
+  , "Negative literals must be wrapped in parenthesis if the negation sign could be mistaken"
```

Comment:
```suggestion
  , "Negative literals must be wrapped in parentheses if the negation sign could be mistaken"
```

Reply:
Oh no... :scream: How long have I been misspelling this!?